roles/deprovision-ios-app/tasks/main.yml

```diff
@@ -1,3 +1,11 @@
 # Delete iOS app representation using the mobile cli
+- set_fact: app_not_found_err="'mobileclients.mobile.k8s.io \"{{ appName }}-ios\" not found'"
```

Comment:
This can be set_fact: app_not_found_err='mobileclients.mobile.k8s.io "{{ appName }}-android" not found'